rest-openehr/src/main/java/org/ehrbase/rest/openehr/audit/OpenEhrAuditStrategy.java

```diff
@@ -0,0 +1,73 @@
+package org.ehrbase.rest.openehr.audit;
```

Comment:
Please add a license header to new files.

Reply:
Done